### ISSUE TITLE
Generated Swift API: No types in responseCompletionHandler

### ISSUE THREAD

**Issue by axelander** [electrode-io/electrode-native#1292]
Trying to use the generated Swift API we just created, but can't seem to get the correct types on the responseCompletionHandler.
```swift
MyApi.init().requests.getMovie(id: 1) { (movie, error) in
  // movie is Any? here, expecting it to be Movie
}
```
Digging around in some tests it looks like I might need to cast this myself.
https://github.com/electrode-io/react-native-electrode-bridge/blob/1b554ef1d61d72b1459255c1b6b4387651d54067/ios/ElectrodeReactNativeBridgeTests/EventProcessorTests.swift#L33-L36

Is this really the only way to get the correct return types? The Kotlin API seems to be able to infer the correct types.

**Comment by axelander** [comment]
I created a PR to get the correct types on the responseCompletionHandler https://github.com/electrode-io/electrode-native/pull/1294

**Comment by friederbluemle** [comment]
According to the discussion, this was addressed in #1294.
